sky/spot/spot_state.py

```diff
@@ -45,12 +75,24 @@
 # total_job_duration = now() - last_recovered_at + job_duration
 # If the job is not finished:
 # total_job_duration = end_at - last_recovered_at + job_duration
-
-_CONN.commit()
 columns = [
```

Comment:
Document: Column names to be displayed e.g., via `sky spot queue`. These may not correspond to actual column names in the DB.

Reply:
Added. Thanks!